src/workspaces/dashboard/AzureStorageDetails.ts

```diff
@@ -37,6 +40,11 @@ export const AzureStorageDetails = (props: AzureStorageDetailsProps): ReactNode
         'aria-label': 'Copy resource group ID to clipboard',
         text: azureContext.managedResourceGroupId,
         style: { marginLeft: '0.25rem' },
+        onClick: (_) => {
```

Comment:
Note that `onClick` doesn't prevent the default action (copying to clipboard) from happening.